spi/control-plane/transfer-spi/src/test/java/org/eclipse/edc/connector/controlplane/transfer/spi/types/TransferProcessTest.java

```diff
@@ -101,6 +101,13 @@ void verifyConsumerTransitions() {
         // should not set the data plane id
         assertThat(process.getDataPlaneId()).isNull();
 
+        process.transitionSuspending("suspension");
+        process.transitionSuspended();
+
+        process.transitionStarted("dataPlaneId");
+        // should not set the data plane id
+        assertThat(process.getDataPlaneId()).isNull();
```

Comment:
these lines are the same as the ones at 100:102.
my proposal would be to extract a (parameterized) test that asserts that the "transitionStarted" method for consumer won't set the dataPlaneId, both coming from `STARTING` and  `SUSPENDED`, so the assertion can be removed from this test (that's already too big, I guess something inherited from the early days of edc)